Decisions.TruCap/Api/DocumentDataResponse.cs

```diff
@@ -207,15 +206,9 @@ public class DocumentDataResponse
         
         public static DocumentDataResponse? JsonDeserialize(string json)
         {
-            try
-            {
-                DocumentDataResponse? text = JsonConvert.DeserializeObject<DocumentDataResponse>(json);
-                return text;
-            }
-            catch (Exception e)
-            {
-                throw new BusinessRuleException(e.Message);
-            }
+            DocumentDataResponse? text = JsonConvert.DeserializeObject<DocumentDataResponse>(json);
+            return text;
```

Comment:
What happens if the nullable `text` variable doesn't have a value? Are we going to return null?

Reply:
I think it should be able to return null